frontend/src/client/services/inquiriesService.ts

```diff
@@ -42,4 +42,17 @@ export function readInquiries(
       422: "Validation Error",
     },
   });
+}
+
+
+export function updateInquiry(updatedInquiry: InquiryPublic): CancelablePromise<InquiryPublic> {
+  return __request(OpenAPI, {
+    method: 'PUT', // Or 'PATCH', depending on your backend implementation
+    url: `/api/v1/inquiries/${updatedInquiry.id}`, // Adjust the endpoint if needed
+    body: updatedInquiry,
+    mediaType: 'application/json',
+    errors: {
+      422: 'Validation Error',
+    },
```

Comment:
Is there a way to get more specific error information?  I was able to relay more specific messages in AddInquiry